main.c

```diff
@@ -705,6 +705,16 @@ static int dev_added(struct tcmu_device *dev)
 	tcmu_dev_dbg(dev, "Got block_size %ld, size in bytes %lld\n",
 		     block_size, dev_size);
 
+	cmd_time_out = tcmu_get_attribute(dev, "cmd_time_out");
+	if (cmd_time_out < 0) {
```

Comment:
Older kernels maybe not have cmd_time_out. If we want to support them, then you need to add a check for -EOPNOTSUPP.

If we do not want to support them, then we should log a more descriptive error message about the device addition failing because the kernel does not support cmd_time_out and also probably advise what kernel they need to upgrade to.

Reply:
Oh yeah, I think for this, we want to add the -EOPNOTSUPP check because glfs just turns it off in their tools if it exists and does not care about changing it.

Reply:
Yes, right.
But I think we should check -ENOENT, because if cmd_time_out is not exist, the open will fail with the errno = ENOENT.

Reply:
@mikechristie @pranithk 
BYW, the glfs tools won't just write zero to cmd_time_out to disable it if it exists ?